documentation/user/admin/UserInterface.rst

```diff
@@ -161,17 +161,17 @@ Each of the software question types make it possible to configure the following
 
   Question: The text of the question
 
-  Hint: A hint that will be shown via an popover an a question mark near the question.
```

Comment:
"an a question mark near the question"
is the part that is neither here nor there.